bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -307,26 +318,44 @@ protected void doConnectedRun() throws IOException, ApiException {
         }
     };
 
+    private final Runnable scenePollingRunnable = new PollingRunnable() {
+        @Override
+        protected void doConnectedRun() throws IOException, ApiException {
+            Map<String, String> groupNames = lastGroupStates.entrySet().parallelStream()
+                    .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().getName()));
+            List<StateOption> stateOptions = hueBridge.getScenes().parallelStream()//
+                    .map(scene -> scene.toStateOption(groupNames))//
+                    .collect(Collectors.toList());
+            stateDescriptionOptionProvider.setStateOptions(new ChannelUID(getThing().getUID(), CHANNEL_SCENE),
+                    stateOptions);
+        }
+    };
+
     private long lightPollingInterval = TimeUnit.SECONDS.toSeconds(10);
     private long sensorPollingInterval = TimeUnit.MILLISECONDS.toMillis(500);
+    private long scenePollingInterval = TimeUnit.MILLISECONDS.convert(10, TimeUnit.MINUTES);
```

Comment:
Why not keeping the interval in minutes with `TimeUnit.MINUTES.toMinutes(10)` ?

Reply:
I find it odd that the user of this field has to implicitly know that this `long` field is defined in minutes, not seconds or milliseconds. Ideally, I think storing all three `long`s as milliseconds would be the most readable option. (The config value for light polling interval has then to be converted later) However, I'm not sure if I that would be out of scope..?

Reply:
For the configuration settings, there is no problem to define a different unit. The unit is displayed to the user. What is important is to offer a unit that makes sense. Milliseconds is really to much detailed when you expect a big value.

But if in the handler, you would like to store internally the 3 values in milliseconds, why not.